misc/scripts/search.sh

```diff
@@ -57,7 +57,7 @@ if [[ $PACKAGE == *@* ]]; then
 			if [[ "$LEN" -eq 0 ]]; then
 				fancy_message warn "There is no package with the name $IRed${PACKAGE%%@*}$NC in the repo $CYAN$REPONAME$NC"
 				error_log 3 "search $PACKAGE@$REPONAME"
-				return 1	
+				exit 1
```

Comment:
When installing multiple packages at the same time, or when upgrading, I think this causes pacstall to stop  instead of continuing to the next package on the list